api/contract/overrides/schemas/TemplateSummaryError.yml

```diff
@@ -1,16 +1,21 @@
 type: object
 properties:
-  kind:
+  message:
     type: string
-  reason:
+  code:
     type: string
-  fields:
+  sources:
     type: array
     items:
       type: string
-  indexes:
+  stackID:
+    type: string
+  summary:
+    $ref: "./TemplateSummaryResources.yml"
+  diff:
+    $ref: "./TemplateSummaryDiff.yml"
+  errors:
     type: array
     items:
-      type: integer
-    x-go-field-type: '[]*int'
-required: [kind, reason, fields, indexes]
+      $ref: "./TemplateSummaryErrors.yml"
+required: [message, code]
```

Comment:
Ignore the messy diff here.. the CLI had a file named `TemplateSummaryError` already that was renamed to `TemplateSummaryErrors`, as OpenAPI now has a `TemplateSummaryError` file

Reply:
Do we still need all the overrides for templates now that this (hopefully) matches the non-override API spec more closely?

Reply:
`influx apply` still heavily depends on these overrides for generating `diff` blocks, as well as some other places for printing out the `resources` block. I hope this can one day be cleaned up, but its outside of the scope of this PR I think